### DISCUSSION TITLE
Make pinned discussions always show up on top of list

### DISCUSSION THREAD

**Discussion by Andre601** [community/community#3597] [Discussions]
The way pinned discussions are displayed is a bit much.
While it sure is colourful and catches the eyes of any possible viewer is it also distracting and doesn't really fit the overall aesthetic usually combined with GitHub (Simple, to the point, not bloated).

The suggestion is simple: Change pinned discussions to actually look like pinned issues do. This would allow to keep a simple and familiar style across the different sections of a repository.

Another idea, if the above wouldn't be accepted for whatever reasons, is to offer color-schemes that aren't this extreme. The ones currently offered are quite extreme contrasts to one another. Either an extreme bright blue or pink or something else. This also can make it harder to read titles of the pinned discussions or even spotting the "icons" in the background.

Take a look at this example taken from [EssentialsX/Essentials/discussions](https://github.com/EssentialsX/Essentials/discussions). Can you see the + icon at the right or left? I barely can.
![image](https://user-images.githubusercontent.com/11576465/117180504-8ae70500-add4-11eb-9b7d-3c82977ad7f3.png)

**Comment by Andre601** [comment]
But something like this does imo not look right.
Especially since such colorful pins are quite out of place for github.

![SmartSelect_20210323-060710_GitHub.jpg](https://user-images.githubusercontent.com/11576465/112096226-26624480-8b9e-11eb-8a1b-720923edfcbb.jpg)

**Comment by PranamBhat** [comment]
I think the current pinned items are looking good. Especially when you have light/dark background theme.

**Comment by Andre601** [comment]
I've updated my original comment with a better explanation to clarify certain things.
